ops/github_telemetry/projects.py

```
@@ -0,0 +1,130 @@
+        "api",
+        "graphql",
+        "-f",
+        f"query={PROJECTS_QUERY}",
+        "-F",
+        f"owner={owner}",
+        "-F",
+        f"repo={repo}",
+        "-F",
+        f"n={max_projects}",
+    ]
+    r = subprocess.run(cmd, capture_output=True, text=True, env=os.environ.copy(), check=False)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)